client/my-sites/stats/stats-list/stats-list-item.jsx

```diff
@@ -210,6 +210,9 @@ module.exports = React.createClass( {
 		}
 
 		switch ( valueData.type ) {
+			case 'raw':
+				value = valueData.value;
+				break;
```

Comment:
This is required to disable the formatting on the right column of the table. Without this, the value would be formatted as a number, e.g. `2,017` instead of `2017`.